frinx-uniconfig/user-guide/operational-procedures/data-security-models/index.md

```diff
@@ -226,14 +227,73 @@ does not already provide encryption capability, then encrypted-paths module is u
 installation of device ('netconfReferenceModuleName' and matching of revisions are ignored).
 !!!
 
+### Change encryption status
+
+!!!
+For proper working of this RPC it is necessary to enable notifications with this parameter:
+
+```settings
+notifications.enabled=true
+```
+!!!
+
+There is one parameter which enable or disable encryption can be changed by 'change-encryption-status' 
+RPC request:
+
+```settings
+crypto.encrypt-enabled=true
+```
+
+Following request is enable encryption for next RPC calls.
```

Comment:
Following request is used to enable encryption:

(encryption happens not only in RPC calls, it also happens in PUT/PATCH/POST calls)

Reply:
done